gateway-sp-comms/src/single_sp.rs

```diff
@@ -814,6 +816,24 @@ impl SingleSp {
             }),
         }
     }
+
+    pub async fn read_rot_cmpa(&self) -> Result<[u8; 512]> {
```

Comment:
One nit and one question:

* Could we add a constant (or possibly two, for naming clarity? I'm ambivalent about this bit) for the CMPA / CFPA size? As written I think clients of this crate will also need to create some `[u8; 512]`s, and constants would probably be friendlier.
* How is a client (MGS, or possibly the client of MGS) supposed to interpret the returned blob? Will hubtools take this as input for some kind of "display what we know about this" or "compare against a known list" operation?

Reply:
I think the interpretation depends on what problems we are actually trying to solve. We can use this with the verify operation in lpc55_sign to validate a particular Hubris image will work with a particular CMPA/CFPA combination. 

The constant is the LPC55 flash page size. We have that defined in a driver but I don't know the best place to put it for other not-LPC55 software. We could also cheat and use the `block_size` call to get the same answer.

Reply:
Sure, added `ROT_PAGE_SIZE` in db88706aa5d6b434ed428712445798b89b8f1ef5.

I agree with @labbott that we'd probably pass the pages into `hubtools` directly, since that's the existing API.